### PR TITLE
Fix tours API output [MA-156]

### PR DESCRIPTION
**Pull request by nikhiltri**

### CODE REVIEW

File: app/Models/Transformers/StopTransformer.php [hunk 1]
```diff
@@ -10,7 +10,7 @@ class StopTransformer extends TransformerAbstract
     public function transform(TwillModelContract $stop): array
     {
         return [
-            'object' => "$stop->object_type:$stop->object_id",
+            'object' => class_basename($stop->selector?->object::class) . ':' . $stop->object_id,
```
Comment (zachgarwood) [inline app/Models/Transformers/StopTransformer.php]: You'll also need to update the `ObjectTransformer`, which currently lowercases the class name:
https://github.com/art-institute-of-chicago/aic-mobile-cms/blob/develop/app/Models/Transformers/ObjectTransformer.php#L21

File: database/migrations/2024_02_27_225737_update_loan_objects_to_is_on_view.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use App\Models\LoanObject;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        LoanObject::query()->update(['is_on_view' => 1]);
```
Comment (zachgarwood) [inline database/migrations/2024_02_27_225737_update_loan_objects_to_is_on_view.php:10]: I think we should also set the default for the `is_on_view` column to true:
```php
$table->boolean('is_on_view')->default(true)->change();
```

**Review by zachgarwood — CHANGES_REQUESTED** [review]
One needed change and one suggested change

File: app/Models/Transformers/StopTransformer.php (hunk 1 above)
Comment (nikhiltri) [reply app/Models/Transformers/StopTransformer.php]: Ah, good to know. I'll update this PR to match what the `ObjectTransformer` is already doing.

**Review by zachgarwood — APPROVED** [review]
🌈 🌟

**Pull request merged**